modules/automatadBidAdapter.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+```
+Module Name:  Automatad Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer:   tech@automatad.com
+```
+
+# Description
+
+Connects to automatad exchange for bids.
+
+automatad bid adapter supports Banner ads.
+
+# Test Parameters
+```
+var adUnits = [
+    {
+        code: 'banner-ad-div',
+        mediaTypes: {
```

Comment:
After sending a request to your endpoint using these settings, the response is missing width and height. Also, all responses should be gzipped.

```
{"id":"0b54ffe2-5c51-4aeb-8bec-e49931f1f08b","seatbid":[{"bid":[{"adm":"        <html amp4ads=\"\" class=\"i-amphtml-inabox i-amphtml-singledoc i-amphtml-standalone i-amphtml-iframed\" i-amphtml-layout=\"\" i-amphtml-no-boilerplate=\"\" amp-version=\"1912050130240\"><head><meta name=\"referrer\" content=\"origin\"><meta http-equiv=\"Content-Security-Policy\" content=\"script-src https://cdn.ampproject.org/;object-src 'none';child-src blob:;frame-src 'none'\"><meta charset=\"utf-8\"><style amp-runtime=\"\" i-amphtml-version=\"011912050130240\">[hidden]{display:none!important}.i-amphtml-element{display:inline-block}.i-amphtml-blurry-placeholder{transition:opacity 0.3s cubic-bezier(0.0,0.0,0.2,1)!important}[layout=nodisplay]:not(.i-amphtml-element){display:none!important}.i-amphtml-layout-fixed,[layout=fixed][width][height]:not(.i-amphtml-layout-fixed){display:inline-block;position:relative}.i-amphtml-layout-responsive,[layout=responsive][width][height]:not(.i-amphtml-layout-responsive),[width][height][sizes]:not(.i-amphtml-layout-responsive){display:block;position:relative}.i-amphtml-layout-intrinsic{display:inline-block;position:relative;max-width:100%}.i-amphtml-intrinsic-sizer{max-width:100%;display:block!important}.i-amphtml-layout-container,.i-amphtml-layout-fixed-height,[layout=container],[layout=fixed-height][height]{display:block;position:relative}.i-amphtml-layout-fill,[layout=fill]:not(.i-amphtml-layout-fill){display:block;overflow:hidden!important;position:absolute;top:0;left:0;bottom:0;right:0}.i-amphtml-layout-flex-item,[layout=flex-item]:not(.i-amphtml-layout-flex-item){display:block;position:relative;-ms-flex:1 1 auto;flex:1 1 auto}.i-amphtml-layout-fluid{position:relative}.i-amphtml-layout-size-defined{overflow:hidden!important}.i-amphtml-layout-awaiting-size{position:absolute!important;top:auto!important;bottom:auto!important}i-amphtml-sizer{display:block!important}.i-amphtml-blurry-placeholder,.i-amphtml-fill-content{display:block;height:0;max-height:100%;max-width:100%;min-height:100%;min-width:100%;width:0;margin:auto}.i-amphtml-layout-size-defined .i-amphtml-fill-content{position:absolute;top:0;left:0;bottom:0;right:0}.i-amphtml-layout-intrinsic .i-amphtml-sizer{max-width:100%}.i-amphtml-replaced-content,.i-amphtml-screen-reader{padding:0!important;border:none!important}.i-amphtml-screen-reader{position:fixed!important;top:0px!important;left:0px!important;width:4px!important;height:4px!important;opacity:0!important;overflow:hidden!important;margin:0!important;display:block!important;visibility:visible!important}.i-amphtml-screen-reader~.i-amphtml-screen-reader{left:8px!important}.i-amphtml-screen-reader~.i-amphtml-screen-reader~.i-amphtml-screen-reader{left:12px!important}.i-amphtml-screen-reader~.i-amphtml-screen-reader~.i-amphtml-screen-reader~.i-amphtml-screen-reader{left:16px!important}.i-amphtml-unresolved{position:relative;overflow:hidden!important}.i-amphtml-select-disabled{-webkit-user-select:none!important;-moz-user-select:none!important;-ms-user-select:none!important;user-select:none!important}.i-amphtml-notbuilt,[layout]:not(.i-amphtml-element){position:relative;overflow:hidden!important;color:transparent!important}.i-amphtml-notbuilt:not(.i-amphtml-layout-container)>*,[layout]:not([layout=container]):not(.i-amphtml-element)>*{display:none}.i-amphtml-ghost{visibility:hidden!important}.i-amphtml-element>[placeholder],[layout]:not(.i-amphtml-element)>[placeholder]{display:block}.i-amphtml-element>[placeholder].amp-hidden,.i-amphtml-element>[placeholder].hidden{visibility:hidden}.i-amphtml-element:not(.amp-notsupported)>[fallback],.i-amphtml-layout-container>[placeholder].amp-hidden,.i-amphtml-layout-container>[placeholder].hidden{display:none}.i-amphtml-layout-size-defined>[fallback],.i-amphtml-layout-size-defined>[placeholder]{position:absolute!important;top:0!important;left:0!important;right:0!important;bottom:0!important;z-index:1}.i-amphtml-notbuilt>[placeholder]{display:block!important}.i-amphtml-hidden-by-media-query{display:none!important}.i-amphtml-element-error{background:red!important;color:#fff!important;position:relative!important}.i-amphtml-element-error:before{content:attr(error-message)}i-amp-scroll-container,i-amphtml-scroll-container{position:absolute;top:0;left:0;right:0;bottom:0;display:block}i-amp-scroll-container.amp-active,i-amphtml-scroll-container.amp-active{overflow:auto;-webkit-overflow-scrolling:touch}.i-amphtml-loading-container{display:block!important;pointer-events:none;z-index:1}.i-amphtml-notbuilt>.i-amphtml-loading-container{display:block!important}.i-amphtml-loading-container.amp-hidden{visibility:hidden}.i-amphtml-element>[overflow]{cursor:pointer;position:relative;z-index:2;visibility:hidden}.i-amphtml-element>[overflow].amp-visible{visibility:visible}template{display:none!important}.amp-border-box,.amp-border-box *,.amp-border-box :after,.amp-border-box :before{box-sizing:border-box}amp-pixel{display:none!important}amp-analytics,amp-story-auto-ads{position:fixed!important;top:0!important;width:1px!important;height:1px!important;overflow:hidden!important;visibility:hidden}html.i-amphtml-fie>amp-analytics{position:initial!important}[visible-when-invalid]:not(.visible),amp-list [fetch-error],form [submit-error],form [submit-success],form [submitting]{display:none}amp-accordion{display:block!important}amp-accordion>section{float:none!important}amp-accordion>section>*{float:none!important;display:block!important;overflow:hidden!important;position:relative!important}amp-accordion,amp-accordion>section{margin:0}amp-accordion>section>:last-child{display:none!important}amp-accordion>section[expanded]>:last-child{display:block!important}\n/*# sourceURL=/css/ampshared.css*/html.i-amphtml-inabox{width:100%!important;height:100%!important}</style><style amp-extension=\"amp-form\">form.amp-form-submit-error [submit-error],form.amp-form-submit-success [submit-success],form.amp-form-submitting [submitting]{display:block}textarea[autoexpand]:not(.i-amphtml-textarea-max){overflow:hidden!important}.i-amphtml-textarea-clone{visibility:hidden;position:absolute;top:-9999px;left:-9999px;height:0!important}.i-amphtml-validation-bubble{transform:translate(-50%,-100%);background-color:#fff;box-shadow:0 5px 15px 0 rgba(0,0,0,0.5);max-width:200px;position:absolute;display:block;box-sizing:border-box;padding:10px;border-radius:5px}.i-amphtml-validation-bubble:after{content:\" \";position:absolute;bottom:-8px;left:30px;width:0;height:0;border-left:8px solid transparent;border-right:8px solid transparent;border-top:8px solid #fff}[visible-when-invalid]{color:red}\n/*# sourceURL=/extensions/amp-form/0.1/amp-form.css*/</style><style amp-extension=\"amp-fit-text\">.i-amphtml-fit-text-content,.i-amphtml-fit-text-content.i-amphtml-fill-content{display:block;display:-ms-flexbox;display:flex;-ms-flex-direction:column;flex-direction:column;-ms-flex-wrap:nowrap;flex-wrap:nowrap;-ms-flex-pack:center;justify-content:center}.i-amphtml-fit-text-content-overflown{display:block;display:-webkit-box;-webkit-box-orient:vertical;overflow:hidden}\n/*# sourceURL=/extensions/amp-fit-text/0.1/amp-fit-text.css*/</style><meta content=\"width=device-width,minimum-scale=1,initial-scale=1\" name=\"viewport\"><meta content=\"vendor=doubleclick,type=impression-id,value=COW7ueuvvOYCFciGjwodUAoLXw\" name=\"amp4ads-id\"><link as=\"script\" href=\"https://cdn.ampproject.org/rtv/011912050130240/amp4ads-v0.js\" rel=\"preload\"><script async=\"\" src=\"https://cdn.ampproject.org/rtv/011912050130240/amp4ads-v0.js\"></script><script async=\"\" custom-element=\"amp-ad-exit\" src=\"https://cdn.ampproject.org/rtv/011912050130240/v0/amp-ad-exit-0.1.js\"></script><script async=\"\" custom-element=\"amp-analytics\" src=\"https://cdn.ampproject.org/rtv/011912050130240/v0/amp-analytics-0.1.js\"></script><script async=\"\" custom-element=\"amp-fit-text\" src=\"https://cdn.ampproject.org/rtv/011912050130240/v0/amp-fit-text-0.1.js\"></script><script async=\"\" custom-element=\"amp-form\" src=\"https://cdn.ampproject.org/rtv/011912050130240/v0/amp-form-0.1.js\"></script><style amp-custom=\"\">a { color: #000000 }body { margin: 0; background: transparent; }#google_image_div {height: 600px;width: 300px;overflow:hidden;position:relative}html, body {width:100%;height:100%;}body {display:table;text-align:center;}#google_center_div {display:table-cell;vertical-align:middle;}#google_image_div {display:inline-block;}.abgc {position:absolute;z-index:2147483646;right:0;top:0;}.abgc amp-img, .abgc img {display:block;}.abgs {position:absolute;-webkit-transform:translateX(97px);transform:translateX(97px);right:16px;top:0;}.abgcp {position:absolute;right:0;top:0;width:31px;height:15px;padding-left:10px;padding-bottom:10px;}.abgb {position:relative;margin-right:16px;top:0;}.abgc:hover .abgs {-webkit-transform:none;transform:none;}.cbb {display: block;position: absolute;right:0;top:0;cursor: pointer;height: 15px;width: 15px;z-index: 9020;padding-left:16px;}.btn {display: inline-block;border-radius: 2px;-moz-box-sizing: border-box;-webkit-box-sizing: border-box;box-sizing: border-box;box-shadow: 0px 0px 2px rgba(0,0,0,0.12), 0px 1px 3px rgba(0,0,0,0.26);cursor: pointer;font-size: 0.7em;margin: 0 1px 0.4em 1px;}@media (max-width: 375px) and (min-height: 100px) {.btn {display: block;width: 90%;max-width: 240px;margin-left: auto;margin-right: auto;}}#spv1 amp-fit-text>div {-webkit-justify-content: flex-start;justify-content: flex-start;}.jt .pn amp-fit-text>div {-webkit-justify-content: flex-start;justify-content: flex-start;}.btn > span {display: inline-block;padding: 0.5em 0.6em;line-height: 1em;}#sbtn {background-color: #FFFFFF;color: #9E9EA6;text-decoration: none;}#sbtn:hover,#sbtn:active {background-color: #F5F5F5;}#rbtn {background-color: rgb(66,133,245);color: white;}#rbtn:hover,#rbtn:active {background-color: #3275E5;}#mta {position:absolute;top: 0;left: 0;font-family: Arial, sans-serif;font-size: 12px;font-weight: 400;line-height: 1em;}#mta input[type=\"radio\"] {display: none;}#mta .pn {right: -300px;top: -600px;width:300px;height:600px;position: absolute;-moz-box-sizing: border-box;-webkit-box-sizing: border-box;box-sizing: border-box;background-color: #FAFAFA;text-align: center;}#spv2 {display: -webkit-flex;display: flex;-webkit-justify-content: flex-start;justify-content: flex-start;-webkit-flex-wrap: nowrap;flex-wrap: nowrap;overflow: hidden;background-color: #FAFAFA;font-size: 0;}.sv #spv2 {-webkit-flex-direction: column;flex-direction: column;}.jt.sv #spv2 {-webkit-justify-content: flex-start;justify-content: flex-start;-webkit-align-items: center;align-items: center;}#spv2 * {-moz-box-sizing: border-box;-webkit-box-sizing: border-box;box-sizing: border-box;}#mta input[name=\"a\"]:checked ~ #cbb {display: none;}#spv3 {opacity:1;}.amp-animate #spv4 {opacity:0;transition: opacity 0.5s linear 2.5s;}.amp-animate  #spv3 amp-fit-text {opacity:1;transition: opacity 0.5s linear 2s;}#spr3:checked ~ #spv3 amp-fit-text {opacity:0}#spr3:checked ~ #spv4 {opacity:1;}#spr1:checked ~ #spv1,#spr2:checked ~ #spv2,#spr3:checked ~ #spv3,#spr3:checked ~ #spv4{right: 0px;top: 0px;}.ct svg {border: 0;margin: 0 0 -0.45em 0;display: inline-block;height: 1.38em;opacity: 0.4;}.ct {display: inline-block;line-height: 1.28em;color: rgba(0,0,0,0.4);text-align:center;padding: 0.3em;}.fct {padding: 1em;}#pct {display: block;font-weight: bold;padding: 1em 0.3em;}#ti {width: 300px;}#btns {width: 300px;}.fl {width: 300px;height:600px;}#si {position: relative;display: inline-block;margin-bottom: -0.15em;height: 1em;width: 1em;opacity: 0.4;}.sb {flex-shrink: 0;height: 50px;}.so {position: relative;z-index: 9110;overflow: hidden;display: inline-block;padding: 1px 5px;width: 96px;height: 50px;border: 1px solid #E0E0E0;background-color: #FFFFFF;cursor: pointer;}.so:hover,.so:active {background-color: #F5F5F5;}.so div {display: -webkit-flex;display: flex;-webkit-align-items: center;align-items: center;-webkit-justify-content: center;justify-content: center;width: 100%;height: 100%;}.so span {color: #4285F4;font-family: Arial, sans-serif;text-align: center;font-size: 12px;line-height: 14px;white-space: normal;}@media (min-height: 54px) {.sh.ss .so,.sv .so {box-shadow: 0px 0px 2px rgba(0,0,0,0.12), 0px 1px 3px rgba(0,0,0,0.26);border: none;}}.sv .so,.sh.ss .so {border-radius: 2px;}.sv .so {margin: 4px;}.sv.jt .so:first-child {margin-top: 8px;}</style></head><body class=\"amp-animate amp-mode-mouse\" marginwidth=\"0\" marginheight=\"0\" style=\"opacity: 1; visibility: visible; animation: 0s ease 0s 1 normal none running none;\"><div id=\"google_center_div\"><div id=\"google_image_div\"><a href=\"https://www.googleadservices.com/pagead/aclk?sa=L&amp;ai=Cb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM&amp;ae=1&amp;num=1&amp;cid=CAASEuRoJTkuau1oSUcYYRZY447JrA&amp;sig=AOD64_2pQNRstDG8gL3vPFEsMTxcrNFsYQ&amp;client=ca-pub-8648432769746717&amp;nx=CLICK_X&amp;ny=CLICK_Y&amp;nb=2&amp;adurl=https://welcome.linode.com/get-started-vps-2019/%3Futm_source%3Dgoogle%26utm_medium%3Dcpc%26utm_campaign%3DPayPerConversion%2520-%2520Remarketing%2520-%2520GDN%2520(LPs%2520%26%2520Home%2520Page)%2520-%2520Asia%2520-%2520DC%26utm_term%3D%26utm_content%3D385740012364%26locationid%3D9062008%26gclid%3DEAIaIQobChMI5bu566-85gIVyIaPCh1QCgtfEAEYASAAEgJlAvD_BwE\" id=\"aw0\" on=\"tap:exit-api.exit(target='landingPage',_nb='2')\" target=\"_blank\"><amp-img alt=\"\" class=\"img_ad i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout\" height=\"600\" i-amphtml-layout=\"fixed\" i-amphtml-ssr=\"\" src=\"https://tpc.googlesyndication.com/simgad/2994163066255105750?sqp=4sqPyQQ7QjkqNxABHQAAtEIgASgBMAk4A0DwkwlYAWBfcAKAAQGIAQGdAQAAgD-oAQGwAYCt4gS4AV_FAS2ynT4&amp;rs=AOga4qnnAYns4I5fiLGN1MdNVIdsMjVHgA\" style=\"width:300px;height:600px;\" width=\"300\"><img alt=\"\" class=\"i-amphtml-fill-content i-amphtml-replaced-content\" decoding=\"async\" src=\"https://tpc.googlesyndication.com/simgad/2994163066255105750?sqp=4sqPyQQ7QjkqNxABHQAAtEIgASgBMAk4A0DwkwlYAWBfcAKAAQGIAQGdAQAAgD-oAQGwAYCt4gS4AV_FAS2ynT4&amp;rs=AOga4qnnAYns4I5fiLGN1MdNVIdsMjVHgA\"></amp-img></a><div aria-hidden=\"true\" class=\"abgc abgf pen\" dir=\"ltr\" id=\"abgc\"><div class=\"abgcp pea\" id=\"abgcp\"><div class=\"abgs\" id=\"abgs\"><a class=\"abgl\" href=\"https://adssettings.google.com/whythisad?reasons=AB3afGEAAAGQW1tbW251bGwsWzEsOSwyMSwzOF0sbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxXSxbbnVsbCxudWxsLFtudWxsLG51bGwsbnVsbCwiaHR0cHM6Ly9kaXNwbGF5YWRzLWZvcm1hdHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tL2Fkcy9wcmV2aWV3L2NvbnRlbnQuanM_Y2xpZW50PXd0YVx1MDAyNm9iZnVzY2F0ZWRDdXN0b21lcklkPTYyNzQ1MjM1MTFcdTAwMjZjcmVhdGl2ZUlkPTM4NTc0MDAxMjM2NFx1MDAyNnZlcnNpb25JZD0wXHUwMDI2YWRHcm91cENyZWF0aXZlSWQ9MzU2NDk5NTUxMjA0XHUwMDI2aHRtbFBhcmVudElkPXByZXYtMFx1MDAyNmhlaWdodD02MDBcdTAwMjZ3aWR0aD0zMDBcdTAwMjZzaWc9QUNpVkJfeEdrVGFiQ21FUlpITU14YXdFT3laaG1jRkpCQSJdXV1dLFsyLDFdXYI6-j2K6kejFySn6aSZ5xuWuhaK8-ReZRKK_PxIuzz_xRa3knqFqbD6KcJG0uCMCQD7svsLahyuuIp4Rd6cwaXQtSSk76b6TTSDlPZYcCh7lgQP0C1hqYc7ec8tD5Ao1UOyAaIhA4OpC8v2zEEAYUaVYWN76hwD3cmGJW3W1ImE8k4nt40R128PkNRuRfPRMAUrQ73FenviiDR0nTrFkxs5VCCoJH6R-EAQyd76z9Owc-PT-ApVxdDjStB-rW0XejzolhLy2UhMtLZkzBjkrsWZ_z7CSuyWSJt7Ka0CLvt51870N9DDcXlVfTY-JUEtXx8ThynuqMYgA4imUpaFiOI,mfhym0oNOKM2nZOvjlGa0A&amp;source=display&amp;cbt=Pt5KYtxY764I5OediLAKEMmwlYYYGK_cMSIKbGlub2RlLmNvbTIICAUTGN_zBxRCF2NhLXB1Yi04NjQ4NDMyNzY5NzQ2NzE3SAVYAnABggEK0O-nkvr_____AagBAQ&amp;cv=https://googleads.g.doubleclick.net/pagead/conversion/%3Fai%3DCb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM%26sigh%3Dvl07SOXteMQ%26cid%3DCAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg\" id=\"abgl\" target=\"_blank\"><amp-img class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout\" height=\"15\" i-amphtml-layout=\"fixed\" i-amphtml-ssr=\"\" src=\"https://tpc.googlesyndication.com/pagead/images/abg/en.png\" style=\"width:96px;height:15px;\" width=\"96\"><img class=\"i-amphtml-fill-content i-amphtml-replaced-content\" decoding=\"async\" src=\"https://tpc.googlesyndication.com/pagead/images/abg/en.png\"></amp-img></a></div><div class=\"abgb\" id=\"abgb\"><a href=\"https://adssettings.google.com/whythisad?reasons=AB3afGEAAAGQW1tbW251bGwsWzEsOSwyMSwzOF0sbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxXSxbbnVsbCxudWxsLFtudWxsLG51bGwsbnVsbCwiaHR0cHM6Ly9kaXNwbGF5YWRzLWZvcm1hdHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tL2Fkcy9wcmV2aWV3L2NvbnRlbnQuanM_Y2xpZW50PXd0YVx1MDAyNm9iZnVzY2F0ZWRDdXN0b21lcklkPTYyNzQ1MjM1MTFcdTAwMjZjcmVhdGl2ZUlkPTM4NTc0MDAxMjM2NFx1MDAyNnZlcnNpb25JZD0wXHUwMDI2YWRHcm91cENyZWF0aXZlSWQ9MzU2NDk5NTUxMjA0XHUwMDI2aHRtbFBhcmVudElkPXByZXYtMFx1MDAyNmhlaWdodD02MDBcdTAwMjZ3aWR0aD0zMDBcdTAwMjZzaWc9QUNpVkJfeEdrVGFiQ21FUlpITU14YXdFT3laaG1jRkpCQSJdXV1dLFsyLDFdXYI6-j2K6kejFySn6aSZ5xuWuhaK8-ReZRKK_PxIuzz_xRa3knqFqbD6KcJG0uCMCQD7svsLahyuuIp4Rd6cwaXQtSSk76b6TTSDlPZYcCh7lgQP0C1hqYc7ec8tD5Ao1UOyAaIhA4OpC8v2zEEAYUaVYWN76hwD3cmGJW3W1ImE8k4nt40R128PkNRuRfPRMAUrQ73FenviiDR0nTrFkxs5VCCoJH6R-EAQyd76z9Owc-PT-ApVxdDjStB-rW0XejzolhLy2UhMtLZkzBjkrsWZ_z7CSuyWSJt7Ka0CLvt51870N9DDcXlVfTY-JUEtXx8ThynuqMYgA4imUpaFiOI,mfhym0oNOKM2nZOvjlGa0A&amp;source=display&amp;cbt=Pt5KYtxY764I5OediLAKEMmwlYYYGK_cMSIKbGlub2RlLmNvbTIICAUTGN_zBxRCF2NhLXB1Yi04NjQ4NDMyNzY5NzQ2NzE3SAVYAnABggEK0O-nkvr_____AagBAQ&amp;cv=https://googleads.g.doubleclick.net/pagead/conversion/%3Fai%3DCb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM%26sigh%3Dvl07SOXteMQ%26cid%3DCAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg\" target=\"_blank\"><amp-img class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout\" height=\"15\" i-amphtml-layout=\"fixed\" i-amphtml-ssr=\"\" src=\"https://tpc.googlesyndication.com/pagead/images/abg/icon.png\" style=\"width:15px;height:15px;\" width=\"15\"><img class=\"i-amphtml-fill-content i-amphtml-replaced-content\" decoding=\"async\" src=\"https://tpc.googlesyndication.com/pagead/images/abg/icon.png\"></amp-img></a></div></div><amp-analytics class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element\" i-amphtml-layout=\"fixed\" style=\"width:1px;height:1px;\" trigger=\"immediate\" aria-hidden=\"true\" hidden=\"\"><script type=\"application/json\">{\"requests\": {\"reportNoLabelInstance\": \"https://googleads.g.doubleclick.net/pagead/conversion/?ai=Cb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM&sigh=vl07SOXteMQ&cid=CAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg&cid=CAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg&label=${label}\",\"reportLabelInstance\": \"https://googleads.g.doubleclick.net/pagead/conversion/?ai=Cb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM&sigh=vl07SOXteMQ&cid=CAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg&cid=CAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg&label=${label}&label_instance=${labelInstance}&cbt=Pt5KYtxY764I5OediLAKEMmwlYYYGK_cMSIKbGlub2RlLmNvbTIICAUTGN_zBxRCF2NhLXB1Yi04NjQ4NDMyNzY5NzQ2NzE3SAVYAnABggEK0O-nkvr_____AagBAQ\"},\"transport\": {\"beacon\": false,\"xhrpost\": false,\"image\": true},\"triggers\": {\"trackMute\": {\"on\": \"click\",\"selector\": \"label[for='spr1']\",\"request\": \"reportNoLabelInstance\"},\"trackReportAd\": {\"on\": \"click\",\"selector\": \"label[for='spr2']\",\"request\": \"reportLabelInstance\"},\"trackSurveyResponse\": {\"on\": \"click\",\"selector\": \"label[for='spr3']\",\"request\": \"reportLabelInstance\"},\"trackWhyThisAd\": {\"on\": \"click\",\"selector\": \"#sbtn\",\"request\": \"reportLabelInstance\"}}}</script></amp-analytics><form action=\"//g.co\" class=\"sv jt amp-animate abgf pen i-amphtml-form\" id=\"mta\" target=\"_blank\" novalidate=\"\"><input id=\"spr1\" name=\"a\" type=\"radio\"><input id=\"spr2\" name=\"a\" type=\"radio\"><input id=\"spr3\" name=\"a\" type=\"radio\"><label class=\"cbb pea\" data-vars-label=\"user_feedback_menu_interaction\" for=\"spr1\" id=\"cbb\"><svg height=\"15\" width=\"15\" xmlns=\"//www.w3.org/2000/svg\"><path d=\"M0,0l15,0l0,15l-15,0Z\" fill=\"#cdcccc\"></path><path d=\"M3.25,3.25l8.5,8.5M11.75,3.25l-8.5,8.5\" stroke=\"#00aecd\" stroke-width=\"1.25\"></path></svg></label><div class=\"pn abgf\" id=\"spv1\"><amp-fit-text class=\"fl i-amphtml-layout-responsive i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout\" height=\"600\" i-amphtml-layout=\"responsive\" layout=\"responsive\" max-font-size=\"20\" min-font-size=\"8\" width=\"300\"><i-amphtml-sizer style=\"display:block;padding-top:200.0000%;\"></i-amphtml-sizer><div class=\"i-amphtml-fill-content i-amphtml-fit-text-content\" style=\"z-index: 2;\"><div style=\"line-height: 1.15em; font-size: 20px;\"><div id=\"ti\"><span class=\"ct\">Ad closed by <svg fill=\"#000000\" viewBox=\"0 0 60 24\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M7.5 16.29c-4.07 0-7.49-3.31-7.49-7.38S3.43 1.53 7.5 1.53c2.25 0 3.85.88 5.06 2.03l-1.42 1.42c-.87-.81-2.04-1.44-3.64-1.44-2.97 0-5.29 2.4-5.29 5.37s2.32 5.37 5.29 5.37c1.93 0 3.03-.77 3.73-1.48.58-.58.95-1.4 1.1-2.54H7.5V8.24h6.79c.07.36.11.79.11 1.26 0 1.51-.41 3.39-1.75 4.72-1.3 1.35-2.95 2.07-5.15 2.07zm17.75-4.76c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm13.08 0c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm12.58-4.46v8.53c0 3.51-2.07 4.95-4.52 4.95-2.3 0-3.69-1.55-4.21-2.81l1.82-.76c.32.77 1.12 1.69 2.39 1.69 1.57 0 2.54-.97 2.54-2.79v-.68h-.07c-.47.58-1.37 1.08-2.5 1.08-2.38 0-4.56-2.07-4.56-4.74 0-2.68 2.18-4.77 4.56-4.77 1.13 0 2.03.5 2.5 1.06h.07v-.76h1.98zm-1.84 4.48c0-1.67-1.12-2.9-2.54-2.9-1.44 0-2.65 1.22-2.65 2.9 0 1.66 1.21 2.86 2.65 2.86 1.43.01 2.54-1.2 2.54-2.86zm5.89-9.52V16h-2.09V2.03h2.09zm8.49 11.07l1.62 1.08c-.52.77-1.78 2.11-3.96 2.11-2.7 0-4.72-2.09-4.72-4.75 0-2.83 2.03-4.75 4.48-4.75 2.47 0 3.67 1.96 4.07 3.02l.22.54-6.36 2.63c.49.95 1.24 1.44 2.3 1.44 1.07 0 1.81-.53 2.35-1.32zm-4.99-1.71l4.25-1.76c-.23-.59-.94-1.01-1.76-1.01-1.06 0-2.54.93-2.49 2.77z\"></path><path d=\"M0 0h62v24H0z\" fill=\"none\"></path></svg></span></div><div id=\"btns\"><label data-vars-label=\"user_feedback_menu_option\" data-vars-label-instance=\"1\" for=\"spr2\"><a class=\"btn\" id=\"rbtn\"><span>Stop seeing this ad</span></a></label><label><a class=\"btn\" data-vars-label=\"closebutton_whythisad_click\" data-vars-label-instance=\"1\" href=\"https://adssettings.google.com/whythisad?reasons=AB3afGEAAAGQW1tbW251bGwsWzEsOSwyMSwzOF0sbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxXSxbbnVsbCxudWxsLFtudWxsLG51bGwsbnVsbCwiaHR0cHM6Ly9kaXNwbGF5YWRzLWZvcm1hdHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tL2Fkcy9wcmV2aWV3L2NvbnRlbnQuanM_Y2xpZW50PXd0YVx1MDAyNm9iZnVzY2F0ZWRDdXN0b21lcklkPTYyNzQ1MjM1MTFcdTAwMjZjcmVhdGl2ZUlkPTM4NTc0MDAxMjM2NFx1MDAyNnZlcnNpb25JZD0wXHUwMDI2YWRHcm91cENyZWF0aXZlSWQ9MzU2NDk5NTUxMjA0XHUwMDI2aHRtbFBhcmVudElkPXByZXYtMFx1MDAyNmhlaWdodD02MDBcdTAwMjZ3aWR0aD0zMDBcdTAwMjZzaWc9QUNpVkJfeEdrVGFiQ21FUlpITU14YXdFT3laaG1jRkpCQSJdXV1dLFsyLDFdXYI6-j2K6kejFySn6aSZ5xuWuhaK8-ReZRKK_PxIuzz_xRa3knqFqbD6KcJG0uCMCQD7svsLahyuuIp4Rd6cwaXQtSSk76b6TTSDlPZYcCh7lgQP0C1hqYc7ec8tD5Ao1UOyAaIhA4OpC8v2zEEAYUaVYWN76hwD3cmGJW3W1ImE8k4nt40R128PkNRuRfPRMAUrQ73FenviiDR0nTrFkxs5VCCoJH6R-EAQyd76z9Owc-PT-ApVxdDjStB-rW0XejzolhLy2UhMtLZkzBjkrsWZ_z7CSuyWSJt7Ka0CLvt51870N9DDcXlVfTY-JUEtXx8ThynuqMYgA4imUpaFiOI,mfhym0oNOKM2nZOvjlGa0A&amp;source=display&amp;cbt=Pt5KYtxY764I5OediLAKEMmwlYYYGK_cMSIKbGlub2RlLmNvbTIICAUTGN_zBxRCF2NhLXB1Yi04NjQ4NDMyNzY5NzQ2NzE3SAVYAnABggEK0O-nkvr_____AagBAQ&amp;cv=https://googleads.g.doubleclick.net/pagead/conversion/%3Fai%3DCb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM%26sigh%3Dvl07SOXteMQ%26cid%3DCAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg\" id=\"sbtn\" target=\"_blank\"><span>Why this ad?&nbsp;<svg fill=\"#000000\" id=\"si\" viewBox=\"0 0 16 16\" xmlns=\"//www.w3.org/2000/svg\"><path d=\"M7.5,1.5a6,6,0,1,0,0,12a6,6,0,1,0,0,-12m0,1a5,5,0,1,1,0,10a5,5,0,1,1,0,-10ZM6.625,11l1.75,0l0,-4.5l-1.75,0ZM7.5,3.75a1,1,0,1,0,0,2a1,1,0,1,0,0,-2Z\"></path></svg></span></a></label></div></div></div><div style=\"position: absolute; top: 0px; left: 0px; z-index: 1; visibility: hidden; line-height: 1.15em; font-size: 20px;\"><div id=\"ti\"><span class=\"ct\">Ad closed by <svg fill=\"#000000\" viewBox=\"0 0 60 24\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M7.5 16.29c-4.07 0-7.49-3.31-7.49-7.38S3.43 1.53 7.5 1.53c2.25 0 3.85.88 5.06 2.03l-1.42 1.42c-.87-.81-2.04-1.44-3.64-1.44-2.97 0-5.29 2.4-5.29 5.37s2.32 5.37 5.29 5.37c1.93 0 3.03-.77 3.73-1.48.58-.58.95-1.4 1.1-2.54H7.5V8.24h6.79c.07.36.11.79.11 1.26 0 1.51-.41 3.39-1.75 4.72-1.3 1.35-2.95 2.07-5.15 2.07zm17.75-4.76c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm13.08 0c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm12.58-4.46v8.53c0 3.51-2.07 4.95-4.52 4.95-2.3 0-3.69-1.55-4.21-2.81l1.82-.76c.32.77 1.12 1.69 2.39 1.69 1.57 0 2.54-.97 2.54-2.79v-.68h-.07c-.47.58-1.37 1.08-2.5 1.08-2.38 0-4.56-2.07-4.56-4.74 0-2.68 2.18-4.77 4.56-4.77 1.13 0 2.03.5 2.5 1.06h.07v-.76h1.98zm-1.84 4.48c0-1.67-1.12-2.9-2.54-2.9-1.44 0-2.65 1.22-2.65 2.9 0 1.66 1.21 2.86 2.65 2.86 1.43.01 2.54-1.2 2.54-2.86zm5.89-9.52V16h-2.09V2.03h2.09zm8.49 11.07l1.62 1.08c-.52.77-1.78 2.11-3.96 2.11-2.7 0-4.72-2.09-4.72-4.75 0-2.83 2.03-4.75 4.48-4.75 2.47 0 3.67 1.96 4.07 3.02l.22.54-6.36 2.63c.49.95 1.24 1.44 2.3 1.44 1.07 0 1.81-.53 2.35-1.32zm-4.99-1.71l4.25-1.76c-.23-.59-.94-1.01-1.76-1.01-1.06 0-2.54.93-2.49 2.77z\"></path><path d=\"M0 0h62v24H0z\" fill=\"none\"></path></svg></span></div><div id=\"btns\"><label data-vars-label=\"user_feedback_menu_option\" data-vars-label-instance=\"1\" for=\"spr2\"><a class=\"btn\" id=\"rbtn\"><span>Stop seeing this ad</span></a></label><label><a class=\"btn\" data-vars-label=\"closebutton_whythisad_click\" data-vars-label-instance=\"1\" href=\"https://adssettings.google.com/whythisad?reasons=AB3afGEAAAGQW1tbW251bGwsWzEsOSwyMSwzOF0sbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxXSxbbnVsbCxudWxsLFtudWxsLG51bGwsbnVsbCwiaHR0cHM6Ly9kaXNwbGF5YWRzLWZvcm1hdHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tL2Fkcy9wcmV2aWV3L2NvbnRlbnQuanM_Y2xpZW50PXd0YVx1MDAyNm9iZnVzY2F0ZWRDdXN0b21lcklkPTYyNzQ1MjM1MTFcdTAwMjZjcmVhdGl2ZUlkPTM4NTc0MDAxMjM2NFx1MDAyNnZlcnNpb25JZD0wXHUwMDI2YWRHcm91cENyZWF0aXZlSWQ9MzU2NDk5NTUxMjA0XHUwMDI2aHRtbFBhcmVudElkPXByZXYtMFx1MDAyNmhlaWdodD02MDBcdTAwMjZ3aWR0aD0zMDBcdTAwMjZzaWc9QUNpVkJfeEdrVGFiQ21FUlpITU14YXdFT3laaG1jRkpCQSJdXV1dLFsyLDFdXYI6-j2K6kejFySn6aSZ5xuWuhaK8-ReZRKK_PxIuzz_xRa3knqFqbD6KcJG0uCMCQD7svsLahyuuIp4Rd6cwaXQtSSk76b6TTSDlPZYcCh7lgQP0C1hqYc7ec8tD5Ao1UOyAaIhA4OpC8v2zEEAYUaVYWN76hwD3cmGJW3W1ImE8k4nt40R128PkNRuRfPRMAUrQ73FenviiDR0nTrFkxs5VCCoJH6R-EAQyd76z9Owc-PT-ApVxdDjStB-rW0XejzolhLy2UhMtLZkzBjkrsWZ_z7CSuyWSJt7Ka0CLvt51870N9DDcXlVfTY-JUEtXx8ThynuqMYgA4imUpaFiOI,mfhym0oNOKM2nZOvjlGa0A&amp;source=display&amp;cbt=Pt5KYtxY764I5OediLAKEMmwlYYYGK_cMSIKbGlub2RlLmNvbTIICAUTGN_zBxRCF2NhLXB1Yi04NjQ4NDMyNzY5NzQ2NzE3SAVYAnABggEK0O-nkvr_____AagBAQ&amp;cv=https://googleads.g.doubleclick.net/pagead/conversion/%3Fai%3DCb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM%26sigh%3Dvl07SOXteMQ%26cid%3DCAQSPADwy9IZTrNEcErFgssDZGmhI_r7RAijWukiAvtLvJ12Jt9EJ1zrGdvNR59TNrb0LGLbbFcjTKN1sk-9Tg\" id=\"sbtn\" target=\"_blank\"><span>Why this ad?&nbsp;<svg fill=\"#000000\" id=\"si\" viewBox=\"0 0 16 16\" xmlns=\"//www.w3.org/2000/svg\"><path d=\"M7.5,1.5a6,6,0,1,0,0,12a6,6,0,1,0,0,-12m0,1a5,5,0,1,1,0,10a5,5,0,1,1,0,-10ZM6.625,11l1.75,0l0,-4.5l-1.75,0ZM7.5,3.75a1,1,0,1,0,0,2a1,1,0,1,0,0,-2Z\"></path></svg></span></a></label></div></div></amp-fit-text></div><div class=\"pn abgf\" id=\"spv2\"><label class=\"sb so\" data-vars-label=\"mute_survey_option\" data-vars-label-instance=\"3\" for=\"spr3\"><div><span>Ad covered content</span></div></label><label class=\"sb so\" data-vars-label=\"mute_survey_option\" data-vars-label-instance=\"12\" for=\"spr3\"><div><span>Not interested in this ad</span></div></label><label class=\"sb so\" data-vars-label=\"mute_survey_option\" data-vars-label-instance=\"11\" for=\"spr3\"><div><span>Already bought this</span></div></label><label class=\"sb so\" data-vars-label=\"mute_survey_option\" data-vars-label-instance=\"2\" for=\"spr3\"><div><span>Seen this ad multiple times</span></div></label></div><div class=\"pn abgf\" id=\"spv3\"><amp-fit-text class=\"fl i-amphtml-layout-responsive i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout\" height=\"600\" i-amphtml-layout=\"responsive\" layout=\"responsive\" max-font-size=\"17\" min-font-size=\"8\" width=\"300\"><i-amphtml-sizer style=\"display:block;padding-top:200.0000%;\"></i-amphtml-sizer><div class=\"i-amphtml-fill-content i-amphtml-fit-text-content\" style=\"z-index: 2;\"><div style=\"line-height: 1.15em; font-size: 17px;\"><span id=\"pct\">We'll try not to show that ad again</span></div></div><div style=\"position: absolute; top: 0px; left: 0px; z-index: 1; visibility: hidden; line-height: 1.15em; font-size: 17px;\"><span id=\"pct\">We'll try not to show that ad again</span></div></amp-fit-text></div><div class=\"pn abgf\" id=\"spv4\"><amp-fit-text class=\"fl i-amphtml-layout-responsive i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout\" height=\"600\" i-amphtml-layout=\"responsive\" layout=\"responsive\" max-font-size=\"20\" min-font-size=\"8\" width=\"300\"><i-amphtml-sizer style=\"display:block;padding-top:200.0000%;\"></i-amphtml-sizer><div class=\"i-amphtml-fill-content i-amphtml-fit-text-content\" style=\"z-index: 2;\"><div style=\"line-height: 1.15em; font-size: 20px;\"><span class=\"ct fct\">Ad closed by <svg fill=\"#000000\" viewBox=\"0 0 60 24\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M7.5 16.29c-4.07 0-7.49-3.31-7.49-7.38S3.43 1.53 7.5 1.53c2.25 0 3.85.88 5.06 2.03l-1.42 1.42c-.87-.81-2.04-1.44-3.64-1.44-2.97 0-5.29 2.4-5.29 5.37s2.32 5.37 5.29 5.37c1.93 0 3.03-.77 3.73-1.48.58-.58.95-1.4 1.1-2.54H7.5V8.24h6.79c.07.36.11.79.11 1.26 0 1.51-.41 3.39-1.75 4.72-1.3 1.35-2.95 2.07-5.15 2.07zm17.75-4.76c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm13.08 0c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm12.58-4.46v8.53c0 3.51-2.07 4.95-4.52 4.95-2.3 0-3.69-1.55-4.21-2.81l1.82-.76c.32.77 1.12 1.69 2.39 1.69 1.57 0 2.54-.97 2.54-2.79v-.68h-.07c-.47.58-1.37 1.08-2.5 1.08-2.38 0-4.56-2.07-4.56-4.74 0-2.68 2.18-4.77 4.56-4.77 1.13 0 2.03.5 2.5 1.06h.07v-.76h1.98zm-1.84 4.48c0-1.67-1.12-2.9-2.54-2.9-1.44 0-2.65 1.22-2.65 2.9 0 1.66 1.21 2.86 2.65 2.86 1.43.01 2.54-1.2 2.54-2.86zm5.89-9.52V16h-2.09V2.03h2.09zm8.49 11.07l1.62 1.08c-.52.77-1.78 2.11-3.96 2.11-2.7 0-4.72-2.09-4.72-4.75 0-2.83 2.03-4.75 4.48-4.75 2.47 0 3.67 1.96 4.07 3.02l.22.54-6.36 2.63c.49.95 1.24 1.44 2.3 1.44 1.07 0 1.81-.53 2.35-1.32zm-4.99-1.71l4.25-1.76c-.23-.59-.94-1.01-1.76-1.01-1.06 0-2.54.93-2.49 2.77z\"></path><path d=\"M0 0h62v24H0z\" fill=\"none\"></path></svg></span></div></div><div style=\"position: absolute; top: 0px; left: 0px; z-index: 1; visibility: hidden; line-height: 1.15em; font-size: 20px;\"><span class=\"ct fct\">Ad closed by <svg fill=\"#000000\" viewBox=\"0 0 60 24\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M7.5 16.29c-4.07 0-7.49-3.31-7.49-7.38S3.43 1.53 7.5 1.53c2.25 0 3.85.88 5.06 2.03l-1.42 1.42c-.87-.81-2.04-1.44-3.64-1.44-2.97 0-5.29 2.4-5.29 5.37s2.32 5.37 5.29 5.37c1.93 0 3.03-.77 3.73-1.48.58-.58.95-1.4 1.1-2.54H7.5V8.24h6.79c.07.36.11.79.11 1.26 0 1.51-.41 3.39-1.75 4.72-1.3 1.35-2.95 2.07-5.15 2.07zm17.75-4.76c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm13.08 0c0 2.74-2.13 4.75-4.75 4.75s-4.75-2.02-4.75-4.75c0-2.75 2.13-4.75 4.75-4.75s4.75 2 4.75 4.75zm-2.08 0c0-1.71-1.24-2.88-2.67-2.88s-2.67 1.17-2.67 2.88c0 1.69 1.24 2.88 2.67 2.88s2.67-1.18 2.67-2.88zm12.58-4.46v8.53c0 3.51-2.07 4.95-4.52 4.95-2.3 0-3.69-1.55-4.21-2.81l1.82-.76c.32.77 1.12 1.69 2.39 1.69 1.57 0 2.54-.97 2.54-2.79v-.68h-.07c-.47.58-1.37 1.08-2.5 1.08-2.38 0-4.56-2.07-4.56-4.74 0-2.68 2.18-4.77 4.56-4.77 1.13 0 2.03.5 2.5 1.06h.07v-.76h1.98zm-1.84 4.48c0-1.67-1.12-2.9-2.54-2.9-1.44 0-2.65 1.22-2.65 2.9 0 1.66 1.21 2.86 2.65 2.86 1.43.01 2.54-1.2 2.54-2.86zm5.89-9.52V16h-2.09V2.03h2.09zm8.49 11.07l1.62 1.08c-.52.77-1.78 2.11-3.96 2.11-2.7 0-4.72-2.09-4.72-4.75 0-2.83 2.03-4.75 4.48-4.75 2.47 0 3.67 1.96 4.07 3.02l.22.54-6.36 2.63c.49.95 1.24 1.44 2.3 1.44 1.07 0 1.81-.53 2.35-1.32zm-4.99-1.71l4.25-1.76c-.23-.59-.94-1.01-1.76-1.01-1.06 0-2.54.93-2.49 2.77z\"></path><path d=\"M0 0h62v24H0z\" fill=\"none\"></path></svg></span></div></amp-fit-text></div></form></div></div></div><amp-analytics class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element\" i-amphtml-layout=\"fixed\" style=\"width:1px;height:1px;\" trigger=\"immediate\" aria-hidden=\"true\" hidden=\"\"><script type=\"application/json\">{\"transport\": {\"beacon\": false, \"xhrpost\": false},\"requests\": {\"visibility1\": \"https://pagead2.googlesyndication.com/pcs/activeview?xai=AKAOjsughew1ytWrznxuBtY192lTVHT8OfsZ31md5fXAbq2nXDmrdoN1HM8yaWDohLl3feUeHSFCALiX2dB-0oS-fEtteLDNrpxyzaCL1ZApHFmRYJFbFxvSyrKiD_zevCV3mRaYxzj1eYZsBpGzvu_P6w&sai=AMfl-YQhrKP6b3NXL4LVtrBO2Ouv6hDCEOdW-V-5yeORA4FutXO3tfwrXUUDGZTkhUmwIqtGiQshsaAZzp__5gFTxE5WdPnRL3cMlSUJSjUBdPkjEs3X-8CyKlPBFyY_&sig=Cg0ArKJSzDh1U69JCBz3EAE&cid=CAASEuRoJTkuau1oSUcYYRZY447JrA&id=ampim&o=${elementX},${elementY}&d=${elementWidth},${elementHeight}&ss=${screenWidth},${screenHeight}&bs=${viewportWidth},${viewportHeight}&mcvt=${maxContinuousVisibleTime}&mtos=0,0,${maxContinuousVisibleTime},${maxContinuousVisibleTime},${maxContinuousVisibleTime}&tos=0,0,${totalVisibleTime},0,0&tfs=${firstSeenTime}&tls=${lastSeenTime}&g=${minVisiblePercentage}&h=${maxVisiblePercentage}&tt=${totalTime}&r=v&adk=1596361485&avms=ampa\"},\"triggers\": {\"continuousVisible\": {\"on\": \"visible\",\"request\": [\"visibility1\"],\"parentPostMessage\": \"impression-viewable\",\"visibilitySpec\": {\"selector\": \":root\",\"visiblePercentageMin\": 50,\"continuousTimeMin\": 1000}},\"visibilityChanged\": {\"on\": \"visible\",\"parentPostMessage\": \"visibility-changed-${intersectionRatio}\",\"visibilitySpec\": {\"repeat\": true,\"selector\": \":root\",\"visiblePercentageThresholds\": [[0,0],[0,5],[5,10],[10,15],[15,20],[20,25],[25,30],[30,35],[35,40],[40,45],[45,50],[50,55],[55,60],[60,65],[65,70],[70,75],[75,80],[80,85],[85,90],[90,95],[95,100],[100,100]]}},\"iniLoad\": {\"on\": \"ini-load\",\"parentPostMessage\": \"{\"c\":\"sfchannel1\",\"p\":\"{\\\"uid\\\":\\\"1\\\",\\\"width\\\":${scrollWidth},\\\"height\\\":${scrollHeight}}\",\"s\":\"creative_geometry_update\"}\n\"}}}</script></amp-analytics><amp-analytics class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element\" i-amphtml-layout=\"fixed\" style=\"width:1px;height:1px;\" trigger=\"immediate\" aria-hidden=\"true\" hidden=\"\"><script type=\"application/json\">{\"transport\": {\"beacon\": true, \"xhrpost\": false},\"requests\": {\"ampeos\": \"https://pagead2.googlesyndication.com/pcs/activeview?xai=AKAOjsughew1ytWrznxuBtY192lTVHT8OfsZ31md5fXAbq2nXDmrdoN1HM8yaWDohLl3feUeHSFCALiX2dB-0oS-fEtteLDNrpxyzaCL1ZApHFmRYJFbFxvSyrKiD_zevCV3mRaYxzj1eYZsBpGzvu_P6w&sai=AMfl-YQhrKP6b3NXL4LVtrBO2Ouv6hDCEOdW-V-5yeORA4FutXO3tfwrXUUDGZTkhUmwIqtGiQshsaAZzp__5gFTxE5WdPnRL3cMlSUJSjUBdPkjEs3X-8CyKlPBFyY_&sig=Cg0ArKJSzDh1U69JCBz3EAE&cid=CAASEuRoJTkuau1oSUcYYRZY447JrA&id=ampeos&o=${elementX},${elementY}&d=${elementWidth},${elementHeight}&ss=${screenWidth},${screenHeight}&bs=${viewportWidth},${viewportHeight}&mcvt=${maxContinuousVisibleTime}&mtos=0,0,${maxContinuousVisibleTime},${maxContinuousVisibleTime},${maxContinuousVisibleTime}&tos=0,0,${totalVisibleTime},0,0&tfs=${firstSeenTime}&tls=${lastSeenTime}&g=${minVisiblePercentage}&h=${maxVisiblePercentage}&pt=${pageLoadTime}&tt=${totalTime}&rpt=${navTiming(navigationStart,loadEventStart)}&rst=${navTiming(navigationStart)}&r=de&isd=${initialScrollDepth}&msd=${maxScrollDepth}\"},\"triggers\": {\"endOfSession\": {\"on\": \"visible\",\"request\": \"ampeos\",\"visibilitySpec\": {\"reportWhen\": \"documentExit\",\"selector\": \":root\",\"visiblePercentageMin\": 50}}}}</script></amp-analytics><amp-pixel allow-ssr-img=\"\" class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element\" i-amphtml-layout=\"fixed\" i-amphtml-ssr=\"\" src=\"//www.google.com/ads/measurement/l?ebcid=ALh7CaRUQsZSV08xHatpssdB1St95s7VNeVLf3618_b4N8ogdoMnCBvfSa8cy8dDbqLn4i-BH0AkjbeRB4RFXHPEzsd_S9nH3A\" style=\"width:1px;height:1px;\" aria-hidden=\"true\"><img src=\"//www.google.com/ads/measurement/l?ebcid=ALh7CaRUQsZSV08xHatpssdB1St95s7VNeVLf3618_b4N8ogdoMnCBvfSa8cy8dDbqLn4i-BH0AkjbeRB4RFXHPEzsd_S9nH3A\"></amp-pixel><div style=\"bottom:0;right:0;width:300px;height:600px;background:initial;position:absolute;max-width:100%;max-height:100%;pointer-events:none;image-rendering:pixelated;z-index:2147483647;background-image:url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACsAAAAWBAMAAACrl3iAAAAABlBMVEUAAAD+AciWmZzWAAAAAnRSTlMAApidrBQAAAB3SURBVBjTfZABDsAwCALpD/j/a7dRsZgsNVlXIyUnwH+tXdDXHUBNdZLdWQdpunsnLBWjO/N2x8XbJJgkLn6mwY1BAp9Wsa+MXe1b1ixNWxunEmn0UWUtdgbp2QC5Jf0iMxG+V4jM9N83ZsJrUNmbPEFmgrWqXj0OTwXAJLDUzQAAAABJRU5ErkJggg==');\"></div><amp-pixel allow-ssr-img=\"\" class=\"i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element\" i-amphtml-layout=\"fixed\" i-amphtml-ssr=\"\" src=\"https://securepubads.g.doubleclick.net/pagead/adview?ai=CUPPZZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgT-AU_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay4rwtCBeIdYqHIUgSEOyDKwATXp7DBoQLgBAGSBQQIBBgBkgUECAUYBJIFBAgFGBiSBQUIBRioAaAGAoAH8-JuqAeOzhuoB9XJG6gHk9gbqAe6BqgH8tkbqAemvhuoB-zVG9gHAfIHBBCivBzSCAcIgGEQARgd8ggbYWR4LXN1YnN5bi0zNDM2NjI3NzMwNzA3MzgwgAoDyAsB2BMM&amp;sigh=nLNjP8Uq_wU&amp;tpd=AGWhJmv-39zeKt_xuKYpAtB_VF_4kY12VDtBIPsOXO4OIS2N8A\" style=\"width:1px;height:1px;\" aria-hidden=\"true\"><img src=\"https://securepubads.g.doubleclick.net/pagead/adview?ai=CUPPZZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgT-AU_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay4rwtCBeIdYqHIUgSEOyDKwATXp7DBoQLgBAGSBQQIBBgBkgUECAUYBJIFBAgFGBiSBQUIBRioAaAGAoAH8-JuqAeOzhuoB9XJG6gHk9gbqAe6BqgH8tkbqAemvhuoB-zVG9gHAfIHBBCivBzSCAcIgGEQARgd8ggbYWR4LXN1YnN5bi0zNDM2NjI3NzMwNzA3MzgwgAoDyAsB2BMM&amp;sigh=nLNjP8Uq_wU&amp;tpd=AGWhJmv-39zeKt_xuKYpAtB_VF_4kY12VDtBIPsOXO4OIS2N8A\"></amp-pixel><amp-ad-exit class=\"i-amphtml-layout-container i-amphtml-element\" i-amphtml-layout=\"container\" id=\"exit-api\" aria-hidden=\"true\"><script type=\"application/json\">{\"filters\":{},\"options\":{\"startTimingEvent\":\"navigationStart\"},\"targets\":{\"landingPage\":{\"filters\":[],\"finalUrl\":\"https://welcome.linode.com/get-started-vps-2019/?utm_source=google&utm_medium=cpc&utm_campaign=PayPerConversion%20-%20Remarketing%20-%20GDN%20(LPs%20&%20Home%20Page)%20-%20Asia%20-%20DC&utm_term=&utm_content=385740012364&locationid=9062008&gclid=EAIaIQobChMI5bu566-85gIVyIaPCh1QCgtfEAEYASAAEgJlAvD_BwE\",\"trackingUrls\":[\"https://www.googleadservices.com/pagead/aclk?sa=L&ai=Cb_wRZaD4XaX1BMiNvgTQlKz4BdTl-7Na5OediLAKtpAfEAEg16-dc2DlyuWDtA6gAfWckf8DyAEC4AIAqAMByAMIqgSBAk_QT6WAC_3EMM2SpFv1wnDwJZP2fbXE6clHu4XXaUEqhrkC-Kyh7Y9cc-KmKRZMg_hwl6EoY3UInsgQY5ISof9cR8TG9jcvRtOknfKwbqrRLr1at5YuYZ1ORPaoqyW3pupq8-v8stnLUyUOmM9Mv8uhWIE-QY2oMMNKOig6If8PxBkkqW0k_qR28fwWwO8B9ngKpT2m-Q1_uaaSu_G0Koa1d2BUxMVbg30gkpaZHMRHksdX-Ik02NJohetnAc1fvYqZkWs9HHvOxhoZT1Hozv0lrGjCnuvXOtEYVIetFBR8mcPV6aRtXz0AtY0jaay47QlMl3mukZlloRNS5jsb3iPTwATXp7DBoQLgBAGgBgKAB_PibqgHjs4bqAfVyRuoB5PYG6gHugaoB_LZG6gHpr4bqAfs1RuoB-zVG9gHAdIIBwiAYRABGB3yCBthZHgtc3Vic3luLTM0MzY2Mjc3MzA3MDczODCxCVl-IBuLOyaagAoDyAsB2BMM&ae=1&num=1&cid=CAASEuRoJTkuau1oSUcYYRZY447JrA&sig=AOD64_2pQNRstDG8gL3vPFEsMTxcrNFsYQ&client=ca-pub-8648432769746717&nb=_nb&nx=CLICK_X&ny=CLICK_Y&act=1&ri=1&adurl=https://welcome.linode.com/get-started-vps-2019/%3Futm_source%3Dgoogle%26utm_medium%3Dcpc%26utm_campaign%3DPayPerConversion%2520-%2520Remarketing%2520-%2520GDN%2520(LPs%2520%26%2520Home%2520Page)%2520-%2520Asia%2520-%2520DC%26utm_term%3D%26utm_content%3D385740012364%26locationid%3D9062008%26gclid%3DEAIaIQobChMI5bu566-85gIVyIaPCh1QCgtfEAEYASAAEgJlAvD_BwE\"],\"vars\":{\"_nb\":{\"defaultValue\":\"\",\"iframeTransportSignal\":\"\"}}}}}</script></amp-ad-exit></body></html>\n","adomain":["https://automatad.com"],"crid":219950853,"h":null,"id":"bid1","impid":"2507ba2d9546b4","nurl":"http://localhost:4050/ortb2/winnotice?impid=${AUCTION_IMP_ID}&auction=${AUCTION_ID}&price=${AUCTION_PRICE}&bidid=${AUCTON_BID_ID}&bidcur=${AUCTION_CURRENCY}","price":0.82441,"w":null}]}]}
```

Reply:
added width and height in response and gzipped the server response